tools/docker/syzbot/Dockerfile

```diff
@@ -18,7 +18,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y -q \
 	qemu-user \
 	apt-transport-https curl gnupg python-is-python3 \
 	# Needed for building Cuttlefish images.
-	rsync libarchive-tools
+	rsync libarchive-tools crossbuild-essential-amd64 \
```

Comment:
crossbuild-essential-amd64 listed twice

Reply:
This is now also under "# Needed for building Cuttlefish images.".
Perhaps add "# Needed for building gVisor." section. Folks may start wondering why we need stuff if Linux builds fine without it :)

Reply:
crossbuild-essential-arm64 includes g++-aarch64-linux-gnu which we install here too. 

> Folks may start wondering why we need stuff if Linux builds fine without it :)

It may be a good interview question. The Linux kernel doesn't have dependencies on libraries and it needs only a compiler, but gVisor is a user-space application with all consequences.